app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ApplicationRecord
+  VALID_EMAIL_REGEX = Settings.user.email.regex
+
+  validates :name, presence: true, length:
```

Comment:
presence có tác dụng gì bạn

Reply:
presence  dùng  để kiểm tra xem object có tồn tại hoặc chứa khoảng trắng không?. 
như trong trường hợp trên thì " presence " kiểm tra "name" có giá trị , và không chưá toàn khoảng trắng thì trả về "true"